cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -301,6 +301,13 @@ func (p *BicepProvider) Destroy(
 				return
 			}
 
+			asyncContext.SetProgress(&DestroyProgress{Message: "Getting AppConfigurations to purge", Timestamp: time.Now()})
```

Comment:
It looks like the service uses "App Configuration" with a space.  Same for "Key Vault", maybe we should do the same in our user facing messages here?